src/widgets/card/Card.tsx

```diff
@@ -0,0 +1,51 @@
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import { DNode, WNode } from '@dojo/framework/widget-core/interfaces';
+import { ThemedMixin, theme } from '@dojo/framework/widget-core/mixins/Themed';
+import * as css from './Card.m.css';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+
+const recongizedChildWidgets: { [key: string]: string } = {
+	'CardHeader': 'header',
+	'CardIconHeader': 'header',
+	'CardFooter': 'footer'
+};
+
+@theme(css)
+export default class Card extends ThemedMixin(WidgetBase) {
+	private _isRegonizedChild(child: DNode): string | undefined {
```

Comment:
Can we think of a different approach to this, this seems like a bit of a hacky pattern to discriminate widgets.